### PR TITLE
updating favicon, title

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="ChainBridge" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="description" content="Cere Network" />
```
Comment (AndreiNavoichyk) [inline public/index.html]: Let's use `ETH-CERE bridge` here

File: public/index.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Chainbridge</title>
+    <title>Cere Network</title>
```
Comment (AndreiNavoichyk) [inline public/index.html]: Let's use `ETH-CERE bridge` here

File: public/manifest.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
-  "short_name": "Chainbridge",
-  "name": "Chainbridge",
+  "short_name": "Cere",
```
Comment (AndreiNavoichyk) [inline public/manifest.json]: Let's use `ETH-CERE bridge` here

File: public/manifest.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+  "name": "Cere Network",
```
Comment (AndreiNavoichyk) [inline public/manifest.json]: Let's use `ETH-CERE bridge` here

File: public/index.html [hunk 5]
```diff
@@ -5,8 +5,8 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="ChainBridge" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="description" content="Cere Network" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
```
Comment (AndreiNavoichyk) [inline public/index.html:9]: Why you can not reuse `cere-logo`?

Comment (karthikeyan-cere) [reply public/index.html:9]: For mac we have to use seperate file which is generated using https://favicon.io/

File: public/manifest.json (hunk 4 above)
Comment (karthikeyan-cere) [reply public/manifest.json]: updated.

File: public/manifest.json (hunk 3 above)
Comment (karthikeyan-cere) [reply public/manifest.json]: Updated.

File: public/index.html (hunk 2 above)
Comment (karthikeyan-cere) [reply public/index.html]: updated.

File: public/index.html (hunk 1 above)
Comment (karthikeyan-cere) [reply public/index.html]: Updated.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**